client/my-sites/plugins/plugins-list/index.jsx

```diff
@@ -0,0 +1,390 @@
+/**
+ * External dependencies
+ */
+import React, { PropTypes } from 'react';
+import classNames from 'classnames';
+import isEmpty from 'lodash/lang/isEmpty';
+
+/**
+ * Internal dependencies
+ */
+import analytics from 'analytics';
+import acceptDialog from 'lib/accept';
+import DisconnectJetpackDialog from 'my-sites/plugins/disconnect-jetpack/disconnect-jetpack-dialog';
+import PluginItem from 'my-sites/plugins/plugin-item/plugin-item';
+import PluginsActions from 'lib/plugins/actions';
+import PluginsListHeader from 'my-sites/plugins/plugin-list-header';
+import PluginsLog from 'lib/plugins/log-store';
+import PluginNotices from 'lib/plugins/notices';
+import SectionHeader from 'components/section-header';
+
+export default React.createClass( {
+	displayName: 'PluginsList',
+
+	mixins: [ PluginNotices ],
+
+	propTypes: {
+		plugins: PropTypes.array,
+		header: PropTypes.string,
+		isWpCom: PropTypes.bool,
+		sites: PropTypes.object,
+		selectedSite: PropTypes.oneOfType( [ PropTypes.bool, PropTypes.object ] ),
+		pluginUpdateCount: PropTypes.number,
+		isPlaceholder: PropTypes.bool,
+	},
+
+	getInitialState() {
+		return {
+			disconnectJetpackDialog: false,
+			bulkManagement: false,
+			selectedPlugins: {}
+		};
+	},
+
+	componentDidMount() {
+		PluginsLog.on( 'change', this.showDisconnectDialog );
+	},
+
+	componentWillUnmount() {
+		PluginsLog.removeListener( 'change', this.showDisconnectDialog );
+	},
+
+	isSelected( plugin ) {
```

Comment:
since we don't actually care about or use `plugin` here, why not get rid of it through destructuring?

```js
isSelected( { slug } ) {
	return !! this.state.selectedPlugins[ slug ];
}
```